ccp/control.sls

```diff
@@ -47,8 +55,7 @@ ccp_source:
 
 ccp_install:
   cmd.watch:
-  - name: . {{ control.dir.base }}/bin/activate; python setup.py install
-  - cwd: {{ control.dir.base }}/fuel
+  - name: {{ control.dir.base }}/bin/pip install {{ control.dir.base }}/fuel_ccp
```

Comment:
It installs into the virtual environment by pip which goes from it `{{ control.dir.base }}/bin/pip`, because `{{ control.dir.base }}` is a virtual environment.